bundles/org.openhab.ui/web/src/components/item/metadata/item-metadata-alexa.vue

```diff
@@ -1,35 +1,65 @@
 <template>
   <div>
     <div style="text-align:right" class="padding-right" v-if="itemType !== 'Group'">
-      <label @click="toggleMultiple" style="cursor:pointer">Multiple</label> <f7-checkbox :checked="multiple" @change="toggleMultiple" />
+      <label @click="toggleMultiple" style="cursor:pointer">Multiple</label>
+      <f7-checkbox :checked="multiple" @change="toggleMultiple" />
     </div>
-    <f7-list>
-      <f7-list-item :key="classSelectKey"
-                    :title="(multiple) ? 'Alexa Classes' : 'Alexa Class'" smart-select :smart-select-params="{ openIn: 'popup', searchbar: true, closeOnSelect: !multiple, scrollToSelectedItem: true }" ref="classes">
-        <select name="parameters" @change="updateClasses" :multiple="multiple">
+    <f7-list v-if="ready">
+      <f7-list-item
+        :key="classSelectKey"
+        :title="'Alexa Device Type' + (itemType !== 'Group' ? !multiple ? '/Attribute' : '/Attributes' : '')"
```

Comment:
Maybe add some parentheses here in the inner ternary to make it easier to read?

Reply:
Done